File: eventbusdemo/src/main/java/com/example/eventbus/BaseApplication.java
package com.example.eventbus;

import android.app.Application;

import com.alibaba.android.arouter.facade.service.SerializationService;
import com.alibaba.android.arouter.launcher.ARouter;

public class BaseApplication extends Application {

    public static final String DEBUG = "debug";

    @Override
    public void onCreate() {
        super.onCreate();
        ARouter.openLog();     // 打印日志
        ARouter.openDebug();   // 开启调试模式(如果在InstantRun模式下运行，必须开启调试模式！线上版本需要关闭,否则有安全风险)
        ARouter.init(this); // 尽可能早，推荐在Application中初始化
    }

}

File: aroute/src/main/java/com/example/aroute/BaseApplication.kt
package com.example.aroute

import android.app.Application
import androidx.viewbinding.BuildConfig
import com.alibaba.android.arouter.launcher.ARouter

class BaseApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        // 如果功能没有生效，就直接打开debug模式
        ARouter.openDebug()
        ARouter.openLog()
        ARouter.init(this)
    }
}